aisdc/attacks/multiple_attacks.py

```diff
@@ -0,0 +1,203 @@
+"""
+An entry point to run multiple attacks including MIA (worst-case and LIRA)
+and attribute inference attack using a single configuration file
+with multiple attack configuration
+
+"""
+from __future__ import annotations
+
+import argparse
+import json
+import os
+import uuid
+from typing import Any
+
+from aisdc.attacks.attack import Attack
+from aisdc.attacks.attack_report_formatter import (
+    GenerateJSONModule,  # pylint: disable = import-error
+)
+from aisdc.attacks.attribute_attack import (
+    AttributeAttack,  # pylint: disable = import-error
+)
+from aisdc.attacks.likelihood_attack import LIRAAttack  # pylint: disable = import-error
+from aisdc.attacks.target import Target
+from aisdc.attacks.worst_case_attack import (
+    WorstCaseAttack,  # pylint: disable = import-error
+)
+
+
+class MultipleAttacks(Attack):
+    """Class to wrap the MIA and AIA attack codes"""
+
+    def __init__(
+        self,
+        config_filename: str = None,
+        output_filename: str = None,
+    ) -> None:
+        super().__init__()
+        self.config_filename = config_filename
+        self.output_filename = output_filename
+        """Constructs an object to execute a worst case attack.
+
+        Parameters
+        ----------
+        config_filename: str
+            name of the configuration file which has configurations in a single JSON file
+            to support running multiple attacks
+        output_filename: str
+            name of the output JSON file to store outputs generated through multiple attack runs
+        """
+
+    def __str__(self):
+        return "Multiple Attacks (MIA and AIA) given configurations"
+
+    def attack(self, target: Target) -> None:
+        """
+        Runs attacks from a Target object and a target model
+
+        Parameters
+        ----------
+        target: attacks.target.Target
+            target as an instance of the Target class. Needs to have x_train,
+            x_test, y_train and y_test set.
+        """
+        with open(self.config_filename, "r+", encoding="utf-8") as f:
+            file_contents = f.read()
+            if file_contents != "":
+                config_file_data = json.loads(file_contents)
+                for config_obj in config_file_data:
+                    params = config_file_data[config_obj]
+                    attack_name = config_obj.split("-")[0]
+                    attack_obj = None
+                    if attack_name == "worst_case":
+                        attack_obj = WorstCaseAttack(**params)
+                        attack_obj.attack(target)
+                    elif attack_name == "lira":
+                        attack_obj = LIRAAttack(**params)
+                        attack_obj.attack(target)
+                    elif attack_name == "attribute":
```

Comment:
Can you add an else: return some kind of error statement, if the attack_name is not one of the three you test for here?

I can see a situation where we accidentally change the attack name and then get very confused when this fails in a strange way